site/kube-addon.md

```diff
@@ -292,3 +292,65 @@ For example,
             - name: IPALLOC_RANGE
               value: 10.0.0.0/16
 ```
+
+## <a name="operations"></a> Operations
+
+Weave Net can be used without Kubernetes, and if you have done this before, you may be familiar with the `weave` CLI.
+
+Within Kubernetes, even though the network peers are managed via Kubernetes and very few commands from the `weave` CLI should be useful, you may still want to run commands to, for example, confirm Weave Net' status. This can be done in various ways:
+
+1. You could [install the `weave` script](/site/installing-weave.md) and run the same commands you would run without Kubernetes:
```

Comment:
Maybe stress not to run `weave launch`. 
Or does it detect "weave is already running"?